### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by suejinkim20** [stagecc/bdc3-website-public#357]
Suggested Accessibility Features to Implement:
1. Implement a labeled region for the carousel by using a `<section>` element that can be used to define the region and `aria-labelledby` can define the heading that contains the label.
2. Screen reader for datasets and stats panels: the voiceover reads the panel text, including bullet points for the datasets. It might be nice to have a more sentence-like structure for the screen reader to read aloud, such as “BDC provides in-demand datasets, such as COVID-19 studies, TOPMed Parent Studies and Freeze 9 data, and sickle cell disease studies.”
3. When pausing and unpausing the carousel, the screen-reader does not announce that the carousel has been paused.

For a list of accessibility features that have been met, please see: https://docs.google.com/document/d/12fIQp890D8d0AyrnUElkahp0h0CoPqbmxII9qWmTqvI/edit#heading=h.hidp9kbft1pe

**Comment by mbwatson** [comment]
very thorough; thank you for this, @suejinkim20! this seems fairly straight-forward stuff to address.

we should track remediation of these now. would you prefer to tweak the name of this issue and track remediation here, or create a new issue, separate from this "audit" issue?
